sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFileFormat.scala

```diff
@@ -89,6 +89,8 @@ class OrcFileFormat
       job: Job,
       options: Map[String, String],
       dataSchema: StructType): OutputWriterFactory = {
+    DataSourceUtils.verifySchema("ORC", dataSchema)
```

Comment:
Thank you for refactoring the PR, @maropu ! What about using `shortName` instead of string literal "ORC" here? Then, we can have the same line like the following.
```
DataSourceUtils.verifySchema(shortName, dataSchema)
```

Reply:
yea, that's smart. I will update.